unstructured_inference/inference/layout.py

```diff
@@ -397,11 +397,13 @@ def process_file_with_model(
     ocr_mode: str = "entire_page",
     fixed_layouts: Optional[List[Optional[List[TextRegion]]]] = None,
     extract_tables: bool = False,
-    pdf_image_dpi: int = 200,
+    pdf_image_dpi: Optional[int] = None,
 ) -> DocumentLayout:
     """Processes pdf file with name filename into a DocumentLayout by using a model identified by
     model_name."""
 
+    if pdf_image_dpi is None:
+        pdf_image_dpi = 200
```

Comment:
should this get set to 300 if 

    model_name == "chipper"
    
?

Reply:
Incoming PR that does that upstream in `unstructured`.